terraform/environments/prd/aws/main.tf

```diff
@@ -5,19 +5,19 @@ terraform {
       version = "5.11.0"
     }
   }
-  backend "s3" {
-    bucket         = "dos14-tf-state"
```

Comment:
why are you deleting this configuration?

Reply:
Because first of all I copy Sergey solution, I thought I was way behind.
I didnt immediately understand what it was for, now Iam understand after watching 38 lecture, I'll fix it